roles/4-server-options/tasks/main.yml

```diff
@@ -35,25 +34,28 @@
   include_role:
     name: usb-lib
   when: usb_lib_install | bool
-  #tags: usb-lib
-
-- name: OSM-VECTOR-MAPS
-  include_role:
-    name: osm-vector-maps
-  when: osm_vector_maps_install | bool
-  #tags: osm, maps
-
-- name: Configure NGINX
+ 
+ - name: Fully Enable / Configure NGINX (already installed in Stage 3-BASE-SERVER) if 'nginx_enabled' is True
```

Comment:
```suggestion
- name: Fully Enable / Configure NGINX (already installed in Stage 3-BASE-SERVER) if 'nginx_enabled' is True
```

Reply:
I rebased  this is out dated have another look please